### ISSUE TITLE
Release Twisted 8.2

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3540]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3540|
|Type|task|
|Created|2008-11-16 18:44:04Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-8.2.x-3540|

<details><summary>Searchable metadata</summary>

```
trac-id__3540 3540
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch___branches_releases_release_8_2_x_3540 /branches/releases/release-8.2.x-3540
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1226861044000000 1226861044000000
changetime__1230492275000000 1230492275000000
version__None None
owner__ 
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

Please coordinate with itamar to resolve [#3543](https://github.com/twisted/twisted/issues/3543) as well.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix commented|
|-|-|

Ready for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @radix<br>@exarkun set status to `new`|
|-|-|

  1. twisted/conch/topfiles/NEWS - first line of the _Fixes_ section talks about "1789" instead of "[#1789](https://github.com/twisted/twisted/issues/1789)"
  1. It looks like pair, flow, and runner are going to be released, but they have no news file entries.  Are there actually any changes since 8.0.0?
  1. twisted/names/topfiles/NEWS has two _Fixes_ items which refer to [#3342](https://github.com/twisted/twisted/issues/3342).  I guess that's okay, if it was intentional, which I guess it probably was.
  1. twisted/news/topfiles/NEWS has no entries for 8.2.  There _have_ been some minor changes, but I guess nothing worth summarizing.  It might be confusing to not have an 8.2 section in the news file though?  otoh, maybe no one will notice.
  1. twisted/topfiles/NEWS - [#2820](https://github.com/twisted/twisted/issues/2820) is mentioned, but it ended up being re-opened
  1. twisted/internet/_sslverify.py - the extra copyright lines probably aren't necessary.  At the very least, the third one shouldn't be added.

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_6"></a>|@ralphm commented|
|-|-|

Replying to [exarkun](#note_5):
>   6. twisted/internet/_sslverify.py - the extra copyright lines probably aren't necessary.  At the very least, the third one shouldn't be added.

This is a result of the current version of trunk, which has three copyright statements. The [branches/releases/release-8.2.x-3540/twisted/internet/_sslverify.py file](branches/releases/release-8.2.x-3540/twisted/internet/_sslverify.py file) in the branch only has two. I added the last line in my branch and then someone came by and also 'fixed' the copyrights before my branch got merged. I didn't notice that when I merged to trunk.

In any case I believe it should become this:

```python
# Copyright (c) 2005 Divmod, Inc.
# Copyright (c) 2005-2008 Twisted Matrix Laboratories.
# See LICENSE for details.
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_8"></a>|@radix set owner to @exarkun|
|-|-|

Ok, mostly resolved, but I have a couple questions (#2 and #4).

1. Fixed

2. Yeah, there weren't any changes to those projects. Given that we have a unified release procedure, it's actually harder to *not* release new versions of all projects. What do you think we should do? Should I just put "No interesting changes" in their NEWS files?

3. Yeah, it was intentional. I wanted to give them separate lines because they seemed like they were fixing different problems.

4. Yeah, the only changes that I saw were project-wide stuff which I've documented in the core NEWS file. I guess this is a case similar to #2. What do you think?

5. Removed

6. I only see two copyright lines in _sslverify.py right now. Maybe someone fixed it in this branch already? If not, can you paste exactly what you're seeing?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set owner to @radix|
|-|-|

2/4 - Okay.  I wasn't sure, since it seems like for the 8.1 release we did something different.  Adding "No interesting changes" to the relevant subproject news files sounds like the right solution.

6 - When I merge the branch into trunk, the first 6 lines of _sslverify.py are thus:

```
# -*- test-case-name: twisted.test.test_sslverify -*-
# Copyright (c) 2005 Divmod, Inc.
# Copyright (c) 2008 Twisted Matrix Laboratories.
# See LICENSE for details.
# Copyright (c) 2005-2008 Twisted Matrix Laboratories.
# Copyright (c) 2005-2008 Twisted Matrix Laboratories.
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_10"></a>|@radix removed owner|
|-|-|

2/4 : Ok, I added the stub entries.

6: The content in the branch itself doesn't look like that. What the hell, SVN? Well, I can't merge forward to fix it, so we'll just have to remember to resolve that issue when we merge to trunk.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set owner to @radix|
|-|-|

+1

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_12"></a>|@radix set status to `closed`|
|-|-|

(In [25735]) Merge release-8.2.0-3540

Author: radix
Reviewer: exarkun
Fixes: [#3540](https://github.com/twisted/twisted/issues/3540)

NEWS file and version updates from the release of Twisted 8.2.0.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
